src/cpp/redis.cpp

```diff
@@ -416,19 +416,8 @@ CommandReply Redis::run_script(const std::string& key,
     return run(cmd);
 }
 
-/*!
-*   \brief Run a script function in the database using the
-*          specificed input and output tensors in a multi-GPU system
-*   \param name The name associated with the script
-*   \param function The name of the function in the script to run
-*   \param inputs The names of input tensors to use in the script
-*   \param outputs The names of output tensors that will be used
-*                  to save script results
-*   \param offset index of the current image, such as a processor
-*                   ID or MPI rank
-*   \param num_gpus the number of gpus for which the script was stored
-*   \throw RuntimeException for all client errors
-*/
+// Run a script function in the database using the
+// specificed input and output tensors in a multi-GPU system
```

Comment:
'specificed' to 'specified'

Reply:
Another minor editorial comment: "Run a script function in the database with multiple GPUs using the specified input and output tensors"